samples/snippets/revoke_dataset_access.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def revoke_dataset_access(dataset_id, entity_id):
```

Comment:
Is there a fixture for `entity_id`? I don't see one here or in `conftest.py`.

Choices:

1. Add such a fixture, but then you'll need to update `samples/snippets/update_dataset_access.py` as well.
2. Use the same entity ID as in `samples/snippets/update_dataset_access.py` hardcoded in the sample.

I'm okay with either fix, but a slight preference for (1).